test/integration-test/test_lib/test_analysis/test_user.py

```diff
@@ -50,5 +51,65 @@ def test_keywords(self):
         self.assertEqual(expected_sorted_keywords_for_channels, sorted_keywords_for_channels)
```

Comment:
There is one extra tab space at the beginning. Please align.

Reply:
@prasadtalasila There are two tab spaces at the beginning of line 51. The first one is there because of the class UserTest and the second one is for the function test_keywords. Were you referring to something else?